README.md

```diff
@@ -7,6 +7,8 @@ A curated list of examples related to actix.
 * [Roseline](https://github.com/DoumanAsh/roseline.rs) : A personal web site and discord & IRC bot to access simple SQLite database. Demonstrates usage of various actix and actix-web concepts.
 * [Actix Auth Server](https://hgill.io/posts/auth-microservice-rust-actix-web-diesel-complete-tutorial-part-1/) : Auth web micro-service with rust using actix-web - complete tutorial. See code in [examples/simple-auth-server](https://github.com/actix/examples/tree/master/simple-auth-server)
 * [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
+* [lemmy](https://github.com/dessalines/lemmy) : uilding a federated alternative to reddit in rust.
```

Comment:
Should be "Building", and probably just "A federated alternative.."

Reply:
yep :)